pages/community/index.js

```diff
@@ -219,303 +108,288 @@ function CommunityIndexPage() {
-          <div className="w-full md:w-3/4 mr-5 rounded-lg relative shadow-xl bg-tutorialCover bg-cover h-half-screen md:h-full bg-center">
-            <div className="w-full h-full bg-primary-500 opacity-25 absolute rounded-lg" />
+      </div>
+      <div className="mt-10 md:flex justify-between w-full h-half-screen">
+        <div className="h-full rounded-lg shadow-lg bg-roadmapCover bg-cover bg-center relative md:w-[49%]">
+          <a
+            href="https://github.com/asyncapi/community/discussions/513"
+            target="_blank"
+            rel="noreferrer"
+          >
+            <div className="w-full h-full absolute rounded-lg" />
             <div className="w-full h-full flex flex-col justify-end absolute">
-              <div className="bg-primary-400 text-left rounded-b-lg p-5 ">
+              <div className="bg-primary-600 text-left rounded-b-lg p-5 ">
                 <Heading
                   level="h2"
-                  typeStyle="heading-lg"
+                  typeStyle="heading-md"
                   className="text-white"
                 >
-                  Browse our Tutorials
+                  Community Goals 2023
                 </Heading>
                 <Heading
                   level="h2"
-                  typeStyle="body-lg"
-                  textColor="text-gray-900"
-                  className="text-slate-500 text-sm"
+                  typeStyle="body-md"
+                  textColor="text-white"
+                  className="text-sm"
                 >
-                  We have thousands of tutorials covering a wide range of
-                  topics. Jump into our catalog!
+                  Look into the AsyncAPI community building/maintenance goals
+                  for 2023 and help us improve
                 </Heading>
               </div>
             </div>
-          </div>
-          <div className="h-half-screen md:h-full w-full md:w-2/4 md:flex md:flex-col md:justify-around mt-10 md:mt-0">
-            <div className="h-4/6 md:h-beforeHalf rounded-lg shadow-lg bg-usecases bg-cover bg-center relative">
-              <div className="w-full h-full bg-pink-500 opacity-25 absolute rounded-lg" />
+          </a>
+        </div>
+        <div className="h-full cursor-pointer rounded-lg shadow-lg bg-usecases bg-cover bg-center mt-10 relative md:w-[49%] md:mt-0">
+          <Link href="/community/contributors">
+            <div>
+              <div className="w-full h-full absolute rounded-lg" />
               <div className="w-full h-full flex flex-col justify-end absolute">
-                <div className="bg-pink-300 text-left rounded-b-lg p-5 ">
+                <div className="bg-yellow-600 text-left rounded-b-lg p-5 ">
                   <Heading
                     level="h2"
                     typeStyle="heading-md"
                     className="text-white"
                   >
-                    Explore our Use-cases
+                    Go above and beyond
```

Comment:
```suggestion
                    Recognize AsyncAPI OSS contributors
```

so my main problem with this header is that it is **not informative** ... the phrase "go above and beyond" doesn't tell me what this section is about or why it matters